packages/clone-testing/src/core.rs

```diff
@@ -519,7 +519,7 @@ mod test {
     impl Uploadable for MockCw20 {
         fn wasm(_chain: &ChainInfoOwned) -> WasmPath {
             let path = PathBuf::from_str(env!("CARGO_MANIFEST_DIR")).unwrap();
-            let path = path.join("../../artifacts/cw20_base.wasm");
+            let path = path.join("../../../artifacts/cw20_base.wasm");
```

Comment:
Are you sure about this change?[#386](https://github.com/AbstractSDK/cw-orchestrator/issues/386)
Is this even ever used?

Reply:
Nice catch, it's not being used, we don't even have this file. I'll put `unimplemented!()` inside